rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ClearingRequestSpecTest.java

```diff
@@ -244,4 +250,43 @@ public void should_document_get_clearingrequests_by_state() throws Exception {
                         )));
     }
 
+    @Test
+    public void should_document_patch_clearingrequest() throws Exception {
+        ClearingRequest updateClearingRequest = new ClearingRequest()
+                .setClearingTeam("clearing.team@sw60.org")
```

Comment:
Actually the ueseRepository is mocked in such a way that will return the user with this email id which is "clearing.team@sw60.org" and if try to change it to "clearing.team@sw360.org" in my test function then this test is failing, because no such user exist with the email "clearing.team@sw360.org" i.e. I have kept it "clearing.team@sw60.org"